trade-finance-manager-api/src/v1/emails/AIN-MIN-confirmation/generate-email-variables.js

```diff
@@ -0,0 +1,20 @@
+const CONSTANTS = require('../../../constants');
+const gefEmailVariables = require('./gef-email-variables');
+const bssEmailVariables = require('./bss-email-variables');
+
+const generateAinMinConfirmationEmailVars = (deal, facilityLists) => {
+  const { dealType } = deal;
+
+  switch (dealType) {
+    case CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS:
+      return bssEmailVariables(deal, facilityLists);
+
+    case CONSTANTS.DEALS.DEAL_TYPE.GEF:
+      return gefEmailVariables(deal, facilityLists);
+
+    default:
+      return {};
+  };
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/UK-Export-Finance/dtfs2/pullRequest?prid=8344001)
```suggestion
  }
```